docs/1.0. Overview.md

```diff
@@ -20,4 +20,4 @@ There is some [prerequisite knowledge](2.0.%20Prerequisites.md) needed and some
 The use of OAuth 2.0 terminology has been adopted in the IS-10 standard. OAuth 2.0 was originally developed for allowing web apps to access protected resources, where the web app is the Client and the protected resource is the Resource Server. However, the NMOS Node acts as both a Client (when, for instance, registering with the NMOS Registry) AND as a Resource Server (when, for instance, accepting a connection request). For this reason the Node is sometimes referred to as the Client and sometimes as the Resource Server depending on context.
 
 ## Specification Precedence
-This guide is not a substitute for the NMOS Specifications. If there are any inconsistencies between this guide and the specifications, please assume that it is the specification that is correct.
+This guide is not a substitute for the NMOS Specifications. If there are any inconsistencies between this guide and the specifications, unequivocally the specification is correct.
```

Comment:
I felt the language should be more affirmative here

Reply:
Agree. (@jonathanthorpe-sony is moving this section and adding some other words I suggested.)

Reply:
I shall ensure the new section's wording is suitably emphatic.

Reply:
Added to #31